libs/@hashintel/type-editor/src/entity-type-editor/property-list-card/property-row.tsx

```diff
@@ -0,0 +1,260 @@
+import { PropertyType, VersionedUrl } from "@blockprotocol/type-system/slim";
```

Comment:
This file is mostly a straight extraction from property-list-card, with the exception of InheritanceData added as props, which affects the readonly tooltip message shown